src/components/token-page/InstantSwap/index.js

```diff
@@ -26,22 +158,63 @@ const InstantSwap = ({ t }) => {
           <p className="TokenPage__instant-swap__header__subtitle">{t('token.instantSwap.subtitle')}</p>
         </header>
 
-        <div className="TokenPage__instant-swap__main">
-          <div className="TokenPage__instant-swap__main__benefits">
-            <Benefit icon={<LockIcon />} text={t('token.instantSwap.benefits.noSignUp')} />
-            <Benefit icon={<InvisibilityIcon />} text={t('token.instantSwap.benefits.noKYC')} />
-            <Benefit icon={<ClockIcon />} text={t('token.instantSwap.benefits.onlyFewMinutes')} />
-            <Benefit icon={<TokenIcon />} text={t('token.instantSwap.benefits.nineHundredPlusCryptoSupported')} />
-          </div>
-          <div className="TokenPage__instant-swap__main__widget">
-            <iframe
-              title="changenow-widget"
-              id="iframe-widget"
-              src="https://changenow.io/embeds/exchange-widget/v2/widget.html?FAQ=true&amount=1&amountFiat&backgroundColor=272D33&darkMode=true&from=eth&horizontal=false&isFiat=false&lang=en-US&link_id=836b188968aaa8&locales=false&logo=true&primaryColor=4038FF&to=joy&toTheMoon=false"
-              style={{ height: '356px', width: '100%', border: 'none' }}
-            ></iframe>
+        <div className="TokenPage__instant-swap__switch-wrapper">
+          <div className="TokenPage__instant-swap__switch">
+            <div
+              role="presentation"
```

Comment:
Why are you using `role="presentation"` here?
I think this actually should be a `button` and [it's not recommended to use `role="presentation"` for items like that](https://kinaole.co/en/when-to-use-rolepresentation-and-when-it-is-not-recommended/).